src/lua/httpc.lua

```diff
@@ -44,6 +44,12 @@ local encoders = {
     ['application/msgpack'] = function(body, _content_type) return msgpack.encode(body) end,
```

Comment:
Regarding the commit message (2nd commit):

The documentation request doesn't mention the API for adding/changing decoders.